### DISCUSSION TITLE
Seeed Studio XIAO RP2350 with C/C++ SDK | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1582] [Q&A]
# Seeed Studio XIAO RP2350 with C/C++ SDK | Seeed Studio Wiki

Use Pico SDK on XIAO RP2350 

https://wiki.seeedstudio.com/xiao-rp2350-c-cpp-sdk/

**Comment by ochik1** [comment]
Step 4: In Build the Project, an error occurred in "cmake .." after "rm * -fr" in the buld directory.
By using "cmake .. -G"Ninja"", this error was avoided and compilation was successful.
